000-cbdb-sandbox/configs/smoke-test.sh

```diff
@@ -0,0 +1,216 @@
+set -e
```

Comment:
The script is missing a shebang at the top (#!/bin/bash) for proper interpretation by the shell.

Reply:
Additionally, please ensure [ShellCheck](https://github.com/koalaman/shellcheck) has been run on the scripts to catch and address any obvious errors. Focus on addressing issues related to the changes you introduced.

Reply:
BTW: I'm really digging that you introduced this file. GREAT WORK!

Reply:
Added in the commit below. I've never used ShellCheck but really digging it, never considered using a linter for bash in the past for some reason
https://github.com/antoniopetrole/bootcamp/commit/6be8307b3c8f8eedd0de89cf7e34cc6ef74fb36e